source/user-guide/workflow/suggestion.rst

```diff
@@ -3,3 +3,28 @@
 Suggestion for improving metadata content
 #########################################
 
+The metadata editor can be configured to analyse metadata and make suggestions to improve it. Examples of this include exploding a list of comma-separated keywords in a single element into multiple individual elements.
+
+The processes available in a given metadata schema are defined in the schema's ``suggest.xsl`` file, for example https://github.com/metadata101/iso19139.mcp/blob/master/src/main/plugin/iso19139.mcp/suggest.xsl
```

Comment:
Maybe also link to the page which explains how to make a process https://geonetwork-opensource.org/manuals/3.6.x/en/user-guide/workflow/batchupdate-xsl.html?highlight=process#adding-batch-process ?

Reply:
@fxprunayre done- hopefully that's all we need?